### PR TITLE
Add availability question to user research participant brief response

### PR DESCRIPTION
**Pull request by Wynndow**
Part of this story on Pivotal [https://www.pivotaltracker.com/story/show/135599725](https://www.pivotaltracker.com/story/show/135599725)
See complimentary PR on the api here: [https://github.com/alphagov/digitalmarketplace-api/pull/513](https://github.com/alphagov/digitalmarketplace-api/pull/513)
See complimentary PR no the supplier frontend here [https://github.com/alphagov/digitalmarketplace-supplier-frontend/pull/586](https://github.com/alphagov/digitalmarketplace-supplier-frontend/pull/586)

This branch is built off the `nice-to-have` branch ([https://github.com/alphagov/digitalmarketplace-frameworks/tree/nice-to-have](https://github.com/alphagov/digitalmarketplace-frameworks/tree/nice-to-have)) and should not be merged until that has been merged.

Previously suppliers did not need to answer the availability question if applying to a user research participants brief, unlike for specialists or outcomes.

This PR adds that question whilst simultaneously merging the availability question for outcomes, specialists and user research participants into one question, with specific content for each lot.

### CODE REVIEW

File: frameworks/digital-outcomes-and-specialists/questions/brief-responses/availability.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: Earliest start date
+question: When is the earliest {% if brief.lot == "digital-outcomes" %}the team can start?{% elif brief.lot == "digital-specialists" %}the specialist can start work?{% else %}you can recruit participants?{% endif %}
+hint: 'eg 31/12/2017'
```
Comment (idavidmcdonald) [inline frameworks/digital-outcomes-and-specialists/questions/brief-responses/availability.yml]: https://www.pivotaltracker.com/story/show/137360327
See suggested hint date in this pivotal story

**Comment by idavidmcdonald** [comment]
I've had a look at this. Have left one comment about hint start dates which is minor.

Seems to look OK but I'll take another look and give the thumbs after it's been rebased in case of any changes.

**Comment by idavidmcdonald** [comment]
Taken a further look at this. In this pull request we are using `brief.lot` whereas in other brief response questions we have written we have been using `brief.lotSlug`. I'm not quite sure what the difference is and if there is a preferred one but it feels like we should be consistent. Probably worth taking a quick look to find out which is most appropriate or if it matters.

Picture from API response for a brief:
image

**Comment by idavidmcdonald** [comment]
According to the API, `lot` and `lotSlug` contain exactly the same information. Therefore I suggest changing this user research copy to run off `lotSlug` for consistency. https://github.com/alphagov/digitalmarketplace-api/blob/master/app/models.py#L1263

**Comment by Wynndow** [comment]
Yeah, it's weird. I did look at the api and noted that lot and lotslug were the same. Just picked the shorter one!

I'll update to lotslug if we use it elsewhere though

**Comment by TheDoubleK** [comment]
Yep - `lot` is a legacy key from the olden days. Everything should use `lotSlug` these days, but no-one has been bold enough to remove it from the API response yet.

**Comment by idavidmcdonald** [comment]
Thanks Kev! Chris and I were discussing this but this gives us our answer - we should choose `lotSlug`.

**Comment by idavidmcdonald** [comment]
Nice, I'll thumbs this up after the rebase.

**Comment by idavidmcdonald** [comment]
👍

**Pull request merged**